frontend/src/components/collaborator/CollaboratorListItem.vue

```diff
@@ -1,138 +1,88 @@
+      <button-edit
+        v-if="editable"
+        color="primary--text"
+        text
+        class="my-n1 v-btn--has-bg"
+      />
     </v-list-item-action>
   </v-list-item>
 </template>
 
 <script>
-import ApiSelect from '@/components/form/api/ApiSelect.vue'
 import UserAvatar from '@/components/user/UserAvatar.vue'
-import IconButton from '@/components/buttons/IconButton.vue'
-import CollaboratorListItemDeactivate from '@/components/collaborator/CollaboratorListItemDeactivate.vue'
-import { errorToMultiLineToast } from '@/components/toast/toasts'
+import ButtonEdit from '@/components/buttons/ButtonEdit.vue'
 
 export default {
   name: 'CollaboratorListItem',
   components: {
-    ApiSelect,
+    ButtonEdit,
     UserAvatar,
-    IconButton,
-    CollaboratorListItemDeactivate,
   },
   props: {
     collaborator: { type: Object, required: true },
-    disabled: { type: Boolean, default: false },
+    editable: { type: Boolean, default: false },
   },
   data: () => ({
-    resendingEmail: false,
+    roles: {
+      manager: {
+        roleTranslation: 'entity.camp.collaborators.manager',
+        abilitiesTranslation: 'global.collaborationAbilities.manager',
+        icons: ['mdi-eye-outline', 'mdi-pencil-outline', 'mdi-cog-outline'],
+      },
+      member: {
+        roleTranslation: 'entity.camp.collaborators.member',
+        abilitiesTranslation: 'global.collaborationAbilities.member',
+        icons: ['mdi-eye-outline', 'mdi-pencil-outline'],
+      },
+      guest: {
+        roleTranslation: 'entity.camp.collaborators.guest',
+        abilitiesTranslation: 'global.collaborationAbilities.guest',
+        icons: ['mdi-eye-outline'],
+      },
+    },
   }),
   computed: {
-    isLastManager() {
-      if (this.collaborator.status !== 'established') return false
-      if (this.collaborator.role !== 'manager') return false
-      const camp = this.collaborator.camp()
-      return (
-        camp
-          ?.campCollaborations()
-          ?.items?.filter((collaborator) => collaborator.status === 'established')
-          .filter((collaborator) => collaborator.role === 'manager').length <= 1
-      )
-    },
-    isOwnCampCollaboration() {
-      if (!(typeof this.collaborator.user === 'function')) {
-        return false
-      }
-      return this.$store.state.auth.user?.id === this.collaborator.user().id
-    },
-  },
-  methods: {
-    resendInvitation() {
-      this.resendingEmail = true
-      this.api
-        .href(this.api.get(), 'campCollaborations', {
-          id: this.collaborator.id,
-          action: 'resend_invitation',
-        })
-        .then((postUrl) => this.api.patch(postUrl, {}))
-        .catch((e) => this.$toast.error(errorToMultiLineToast(e)))
-        .finally(() => {
-          this.resendingEmail = false
-        })
+    name() {
+      return this.collaborator.user
+        ? this.collaborator.user().displayName
+        : this.collaborator.inviteEmail
```

Comment:
We have helpers for this, e.g. `campCollaborationDisplayName(this.collaborator, this.$tc.bind(this), false)` would do the same and is unit tested.

Reply:
#3501

Reply:
#3501